### ISSUE TITLE
Black banner has black heading

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_scarfolk#16]
![image](https://user-images.githubusercontent.com/3852805/189165585-9093e64f-1897-4142-85e9-2e0513377a3d.png)

**Comment by keelanfh** [comment]
Fixed in https://github.com/localgovdrupal/localgov_base/issues/310
